examples/gradients.rs

```diff
@@ -5,8 +5,11 @@
 //! `cargo run --example gradients ./examples/empire-state-building.jpg ./tmp`
 
 use image::{open, GrayImage};
+#[cfg(not(feature = "rayon"))]
+use imageproc::filter::filter_clamped;
+#[cfg(feature = "rayon")]
+use imageproc::filter::filter_clamped_parallel as filter_clamped;
 use imageproc::{
-    filter::filter_clamped,
```

Comment:
I don't think it's worth adding this sort of indirection to an example. Examples are just meant for teaching new users how to use the library, this complicates things and could potentially teach bad habits. I would just leave the examples using the single-threaded version.

Reply:
Ah yes, makes sense. Removed it.